### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#271] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 131730](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/36d0ec67-ad33-4de6-bf41-67b74d3241b8)

**Comment by Elia-Belli** [comment]
Una base di $W$ può essere $\mathcal{B}=\\{(1,0,0,-1),(1,0,-1,0),(0,1,0,0)\\}$, quindi $dim\ W=3$

Un supplementare di $W$ ha dimensione $1$ in $\mathbb{R}^4$, quindi ci basta trovare un vettore che non appartenga allo $Span(\mathcal{B})$: siccome $W$ è un sottospazio tutte le comb. lineari dei vettori della base mantengono la proprietà $x_1+x_3+x_4=0$, ci basta prendere un vettore di $\mathbb{R}^4$ che non la rispetta, quindi $U=Span((1,0,0,0))$, oppure $U'=Span((0,0,0,1))$

**Comment by daromacarlo** [comment]
![Immagine 2024-01-17 191954](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/5f3353da-120d-4952-803f-5d4c73b3db5e)
![Immagine 2024-01-17 192007](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/050cf66e-5b97-455b-9835-0b149451419a)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/3c5cb9bf-608f-40c9-a352-cdc8c5560448)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4a6974a8-559b-4098-bae5-2519f21de084)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/afcb01e1-6d04-4cf1-b764-45a6358717a8)
